src/js/modal-dialog.js

```diff
@@ -399,6 +410,94 @@ class ModalDialog extends Component {
     }
     return this.content_;
   }
+
+  /**
+   * conditionally focus the modal dialog if focus was previously on the player.
+   *
+   * @private
+   */
+  conditionalFocus_() {
+    const activeEl = document.activeElement;
+    const playerEl = this.player_.el_;
```

Comment:
I am wondering if we even need a const for this here, since it is only used in the if statement below. Doesn't matter either way though.